pkg/config/config_template.yaml

```diff
@@ -170,6 +170,13 @@ api_key:
 #
 # forwarder_retry_queue_max_size: 30
 
+## @param forwarder_retry_queue_all_payloads_max_size - integer - optional - default: 0
+## It defines the maximum size in bytes of all the payloads in the forwarder's retry queue.
+## If "forwarder_retry_queue_all_payloads_max_size" is greater than 0, 
+## "forwarder_retry_queue_max_size" is ignored.
+#
+# forwarder_retry_queue_all_payloads_max_size: 0
+
```

Comment:
Can we add a sentence about the option being ignored if this is set to `forwarder_retry_queue_max_size`?

Reply:
What do you mean? There is already a sentence explaining what happen when both settings are set.
`If "forwarder_retry_queue_all_payloads_max_size" is greater than 0, "forwarder_retry_queue_max_size" is ignored.`

Reply:
I meant there is this sentence on this new config, but we should add the same sentence to the old config to avoid confusion.